.husky/commit-msg

```diff
@@ -0,0 +1 @@
+npx commitlint --edit $1
```

Comment:
Not a blocker, but I really really prefer all files to end with a newline.

Reply:
I mean... I guess this should be solved by a tool (Prettier? another one?) rather than me manually adding the newline on each file 😅 . Having said that, I think the other repos we're using in the js team do not have the newline at the end. I'd rather be consistent with us and keep these as they are, and then if you feel like once Gabriel is back, we can settle once and for all whether to finish files with a newline or not, and automate it

Reply:
That's why I didn't make this a blocker.  And we should definitely be consistent.

I'd argue in the future that our standard should be newline at the end of all files (unless it is some data file that has a reason for no newline of course).